var/spack/repos/builtin/packages/gcc-runtime/package.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import re
+
+from macholib import MachO, mach_o
+
+from spack.package import *
+from spack.util.elf import parse_elf
+
+
+class GccRuntime(Package):
+    """Package for GCC compiler runtime libraries"""
+
+    homepage = "https://gcc.gnu.org"
+    has_code = False
+
+    tags = ["runtime"]
+
+    maintainers("haampie")
+
+    license("GPL-3.0-only")
```

Comment:
This is wrong, both for `gcc` (which is `GPL-3.0-or-later`) and for the runtime library, which is `GPL-3.0-or-later WITH GCC-exception-3.1`.

```suggestion
    license("GPL-3.0-or-later WITH GCC-exception-3.1")
```